classes/services/PKPSubmissionFileService.inc.php

```diff
@@ -0,0 +1,837 @@
+<?php
+/**
+ * @file classes/services/PKPSubmissionFileService.php
+ *
+ * Copyright (c) 2014-2019 Simon Fraser University
+ * Copyright (c) 2000-2019 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
```

Comment:
Watch out for old code headers! This introduces a 2019 end date, plus a GPLv2 license.